### PR TITLE
Enumerable methods

### PR DESCRIPTION
**Pull request by Mujeeb4582**
# Task completed
- Add the myList class in a separate file
- Add the module in a separate file
- Add the required three methods
- Add the professional readme according to requirements 
- Add the linters setup

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+list.all? { |e| e < 5 }
+
+list.all? { |e| e > 5 }
+
+list.any? { |e| e == 2 }
+
+list.any? { |e| e == 5 }
+
+list.filter(&:even?)
```
Comment (bobb-Rob) [inline my_list.rb]: - [x] [REQUIRED] Great work you have done so far. I noticed you have not required the file `myenum.rb` in your `my_list` class. 
at the top of the `my_list.rb` file, include the code below: 
```
require_relative './myenum.rb'
```
See below the error I am getting when I run your project. 
image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Mujeeb4582 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: my_list.rb (hunk 1 above)
Comment (Mujeeb4582) [reply my_list.rb]: @bobb-Rob Thanks for finding error

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: my_list.rb (hunk 1 above)
Comment (Trust-007) [reply my_list.rb]: Thanks @bobb-Rob

**Pull request merged**